workers/unity/Packages/com.improbable.gdk.core/Attributes/RemoveAtEndOfTick.cs

```diff
@@ -4,11 +4,13 @@
 namespace Improbable.Gdk.Core
 {
     /// <summary>
-    ///     Any component with this attribute will be removed from all entities by the CleanReactiveComponentSystem
+    ///     Any component that inheriting this interrace will be removed from all entities
```

Comment:
typo: interface

Reply:
Why split over two lines?
Also grammar point - "Any component which inherits this interface"